perses/tests/test_geometry_engine.py

```diff
@@ -26,14 +26,553 @@
+        self._default_positions[1] = unit.Quantity(np.array([ 0.0765,  0.1  ,  -0.4005]), unit=unit.nanometers)
+        self._default_positions[2] = unit.Quantity(np.array([ 0.0829 , 0.0952 ,-0.2479]) ,unit=unit.nanometers)
+        self._default_positions[3] = unit.Quantity(np.array([-0.057 ,  0.0951 ,-0.1863] ) ,unit=unit.nanometers)
+
+        self._default_r0 = unit.Quantity(value=0.1522, unit=unit.nanometer)
+        self._default_bond_k = unit.Quantity(value=265265.60000000003, unit=unit.kilojoule/(unit.nanometer**2*unit.mole))
+        self._default_angle_theta0 = unit.Quantity(value=1.91113635, unit=unit.radian)
+        self._default_angle_k = unit.Quantity(value=418.40000000000003, unit=unit.kilojoule/(unit.mole*unit.radian**2))
+        self._default_torsion_periodicity = 2
+        self._default_torsion_phase = unit.Quantity(value=np.pi/2.0, unit=unit.radians)
+        self._default_torsion_k = unit.Quantity(value=20.0, unit=unit.kilojoule/unit.mole)
+
+        self._topology = app.Topology()
+        new_chain = self._topology.addChain("0")
+        new_res = self._topology.addResidue("MOL", new_chain)
+        atom1 = self._topology.addAtom("C1", app.Element.getByAtomicNumber(6), new_res, 0)
+        atom2 = self._topology.addAtom("C2", app.Element.getByAtomicNumber(6), new_res, 1)
+        atom3 = self._topology.addAtom("C3", app.Element.getByAtomicNumber(6), new_res, 2)
+        atom4 = self._topology.addAtom("C4", app.Element.getByAtomicNumber(6), new_res, 3)
+
+
+        self._topology.addBond(atom1, atom2)
+        self._topology.addBond(atom2, atom3)
+        self._topology.addBond(atom3, atom4)
+
+        self._system = openmm.System()
+        indices = [self._system.addParticle(CARBON_MASS) for i in range(4)]
+
+        self._growth_order = [0]
+
+        if bond:
+            bond_force = openmm.HarmonicBondForce()
+            self._system.addForce(bond_force)
+            bond_force.addBond(0, 1, self._default_r0, self._default_bond_k)
+
+        if angle:
+            angle_force = openmm.HarmonicAngleForce()
+            self._system.addForce(angle_force)
+            angle_force.addAngle(0, 1, 2, self._default_angle_theta0, self._default_angle_k)
+
+        if torsion:
+            torsion_force = openmm.PeriodicTorsionForce()
+            self._system.addForce(torsion_force)
+            torsion_force.addTorsion(0, 1, 2, 3, self._default_torsion_periodicity, self._default_torsion_phase, self._default_torsion_k)
+
+        self._structure = parmed.openmm.load_topology(self._topology, self._system)
+        self._positions = self._default_positions
+        self._integrator = openmm.VerletIntegrator(1)
+        self._platform = openmm.Platform.getPlatformByName("Reference")
+        self._context = openmm.Context(self._system, self._integrator, self._platform)
+        self._context.setPositions(self._positions)
+
+    @property
```

Comment:
Should the accessible properties of the class also be documented in a docstring?